cpp/include/cuspatial/detail/range/multilinestring_range.cuh

```diff
@@ -74,6 +74,7 @@ template <typename GeometryIterator, typename PartIterator, typename VecIterator
 class multilinestring_range;
 
 template <typename GeometryIterator, typename PartIterator, typename VecIterator>
+CUSPATIAL_HOST_DEVICE
```

Comment:
clangd was complaining about an invalid host overload of a host device function, so I added this here to make it happy.